public/editor/stylesheets/editor.less

```diff
@@ -14,14 +15,14 @@ body {
 @transitionTime: 0.15s;
 
 #webmaker-bramble {
-    position: absolute;
-    display: block;
-    top: @brambleTop;
-    left: 0;
-    width: 100%;
-    bottom: 0;
-    border: 0;
-    background-color: #f8f8f8;
+  position: absolute;
+  display: block;
+  top: @brambleTop;
+  left: 0;
+  width: 100%;
```

Comment:
rather than `width:100%`, since we're already using left and bottom, can we say `right: 0`?